awscli/autocomplete/db.py

```diff
@@ -33,9 +34,16 @@ def __init__(self, db_filename=None):
     @property
     def _connection(self):
         if self._db_conn is None:
-            self._db_conn = sqlite3.connect(
-                self._db_filename, check_same_thread=False,
-                isolation_level=None)
+            kwargs = {
+                'check_same_thread': False,
+                'isolation_level': None
+            }
+            if self._db_filename.startswith('file::memory:'):
+                # This statement was added because old versions of sqlite don't support 'uri' but we use it for tests
```

Comment:
The comments look longer than 79 characters. Can we break these lines up a little bit to make them shorter?